tests/functional/HelpersCest.php

```diff
@@ -215,4 +216,62 @@ public function itAddsGroupArgumentsToRoutes(FunctionalTester $I)
         $I->assertEquals($route->getDomain(), 'test.pl');
         $I->assertEquals($route->getPrefix(), null);
     }
+
+    public function itSetsRequestTimezoneOnApiHttpHeader(FunctionalTester $I)
+    {
+        // 1st case: no header set
+        $I->sendGet(apiUrl('someapi'));
+
+        $I->assertEquals('UTC', getRequestTimezone());
+
+        // 2nd case: the http header is set
+        $I->haveHttpHeader('Accept-Timezone', 'America/New_York');
+        $I->sendGet(apiUrl('someapi'));
+
+        $I->assertEquals('America/New_York', getRequestTimezone());
+    }
+
+    public function itConvertsDateTimeObjectToRequestTimezone(FunctionalTester $I)
+    {
+        // 1st case: no header set
+        $I->sendGet(apiUrl('someapi'));
+
+        $dateTime = Carbon::parse('2021-05-02 12:43:31', 'America/New_York');
+        $dateTimeInRequestTimezone = dateTimeToRequestTimezone($dateTime);
+
+        $I->assertEquals($dateTime, $dateTimeInRequestTimezone);
```

Comment:
should these two be equals here?

Reply:
These are equal in fact - they two point to the same moment in time.